---
title: "Mickaël Canouil - Curriculum Vitæ"
author: Mickaël Canouil
output:
  pagedown::html_resume:
    css: ["css/curriculum_vitae.css"]
    mathjax: null
---

# Aside

![Mickaël Canouil](pictures/id_portrait_square.png){width=226px, id="picture"}


<!-- > The best thing about being a statistician is that you get to play in everyone’s backyard." -->
<!-- > -- John Tukey -->



## Contact Info {#contact}

- <i class="fa fa-user" style="color: #4169e1;"></i> Head of the Biostatistic Team
- <i class="fa fa-university" style="color: #4169e1;"></i> [EGID - UMR 8199](http://www.good.cnrs.fr/?lang=en)
- <i class="fa fa-map-marker" style="color: #4169e1;"></i> Lille, France
- <i class="fa fa-envelope" style="color: #4169e1;"></i> [mickael.canouil [at] cnrs.fr](mailto:mickael.canouil@cnrs.fr)
- <i class="fa fa-phone" style="color: #4169e1;"></i> +33 (0)3 74 00 81 29
- <i class="fa fa-linkedin" style="color: #4169e1;"></i> [linkedin.com/in/mickaelcanouil](https://www.linkedin.com/in/mickaelcanouil)
- <i class="fa fa-github" style="color: #4169e1;"></i> [github.com/mcanouil](https://github.com/mcanouil)
- <i class="fa fa-twitter" style="color: #4169e1;"></i> [twitter.com/mickaelcanouil](https://twitter.com/mickaelcanouil)
<!-- - <i class="fab fa-docker" style="color: #4169e1;"></i> [hub.docker.com/u/mcanouil](https://hub.docker.com/u/mcanouil) -->
<!-- - <i class="fab fa-reddit" style="color: #4169e1;"s></i> [u/Epimetheos](https://www.reddit.com/user/Epimetheos/) -->

## Computer Skills {#skills}

- <u style="color: #4169e1;">*Advanced:*</u> R (Shiny, Rmarkdown, S4, etc.), HTML, css, LATEX, markdown
- <u style="color: #4169e1;">*Intermediate:*</u> Docker, bash, Julia, Perl, Python, SAS
- <u style="color: #4169e1;">*Basic:*</u> C/C++, SQL, NoSQL, Lua

## Disclaimer {#disclaimer}

Last updated on `r Sys.Date()`.



# Main

## Mickaël Canouil {#title}

### Curriculum Vitæ

I'm currently working for the *Institut Pasteur de Lille* 
at the *UMR 8199 - "Integrated Genomics and Metabolic Diseases Modelling"* 
as the *head of the biostatistic team*.

<!-- #### Main Activities -->

<!-- ##### Data Analysis -->

<!-- I've been analysing omics (SNP, CpG, expression, metabolites, etc.)  -->
<!-- data within numerous projects related to metabolic diseases,  -->
<!-- such as Type 2 Diabetes.  -->
<!-- These projects involve collaborations with national  -->
<!-- and international consortia like CKDgen, CHARGE, IMIDIA,  -->
<!-- DIRECT or more recently RHAPSODY. -->
<!-- My contribution covers Genome-Wide Association Studies (GWAS),  -->
<!-- differential methylation/expression analyses, metabolomics analyses,  -->
<!-- rare variants analyses (clustering approach), disease progression modelling,  -->
<!-- genetic epidemiology and meta-analyses. -->

<!-- ##### Team Management -->

<!-- I've been managing a team of three junior statisticians.  -->
<!-- My role is to provide guidance regarding choices of statistical methodologies  -->
<!-- for analysing the data and code optimisation for implementing these methodologies  -->
<!-- in large scale omics data. -->

<!-- ##### Research -->

<!-- My main research interests are mixed models, more recently extended to joint models. -->
<!-- Joint model, especially the joint likelihood approach implemented in the R package JM  -->
<!-- was studied in the context of Type 2 Diabetes incidence and fasting glucose progression  -->
<!-- (associated by diagnosis definition) using SNPs as biomarkers of interest. -->



## Education {data-icon=graduation-cap data-concise=true}

### Doctor of Philosophy (Ph.D.) in BioStatistics

University of Lille

Lille, France

Oct. 2014 - Sep. 2017

Thesis: *Development and Application of Statistical Methods for Multi-Omics Studies in Type 2 Diabetes: Beyond the Genome-Wide Association Studies Era*

### Master's Degree in Biostatistics, Bioinformatics and Genomics (*Specialised in Biostatistics*)

University Claude Bernard Lyon 1

Lyon, France

Sep. 2009 - Jul. 2011

Internships:

- *Mathematical Modelling of Nosocomial Rotavirus Infections in Pediatric Ward*
- *Development of a Non-Parametric (on Time) Clustering Algorithm for Longitudinal Data*

### Bachelor's Degree in Biology (*Specialised in Biostatistics*)

University Claude Bernard Lyon 1

Lyon, France

Sep. 2006 - Jul. 2009


## Professional & Research Experience {data-icon=laptop}

### Head of the Biostatistic Team

Laboratory of Integrated Genomics  
and Metabolic Diseases Modeling  
EGID - UMR 8199  
(European Genomics Institute for Diabetes)

Lille, France

Jan. 2016 - Present

Activities: *Genome-wide association studies, experimental design, -omics data analysis, methodological developments, team management*

### Biostatistician

Laboratory of Integrated Genomics  
and Metabolic Diseases Modeling  
EGID - UMR 8199  
(European Genomics Institute for Diabetes)

Lille, France

Sep. 2012 - Dec. 2015

Activities: *Genome-wide association studies, experimental design, -omics data analysis, methodological developments*

### Biostatistician

French Institute of Science and Technology for  
Transport, Development and Networks 
(IFSTTAR)  
UMRESTTE - UMR T9405

Bron, France

Nov. 2011 - Dec. 2011

Activities: *Analysis of a mobility and accident study in secondary-school pupils*

### Biostatistician (Internship)

Hospices Civiles de Lyon  
Biostatistics Unit  
UMR CNRS 5558

Lyon, France

Jan. 2011 - Jun. 2011

Activities: *Development of a Non-Parametric (on Time) Clustering Algorithm for Longitudinal Data*

### Biostatistician (Internship)

Biometry and Evolutionary Biology laboratory  
(LBBE)  
UMR CNRS 5558

Lyon, France

Mar. 2010 - Jun. 2010

Activities: *Mathematical Modelling of Nosocomial Rotavirus Infections in Pediatric Ward*


## Teaching Experience {data-icon=chalkboard-teacher}

### R and Databases

Two-days training (in French)  
"Unités Régionales de Formation  
à l'Information Scientifique et Technique"  
(URFIST)

Bordeaux, France

May 2018

::: aside
[<i class="fa fa-github"></i> GitHub](https://github.com/mcanouil/PRESENTATION/tree/5e439ff1db7f031c54f05d2cdcc6fbcfaeb21c32/rdatabase)
:::

### Julia for Intensive Scientific Computing

Half-day workshop (in French)  
"Journées nationales du  
DEVeloppement logiciel"  
(JDEV)

Bordeaux, France

Jul. 2015

::: aside
[<i class="fa fa-github"></i> GitHub](https://github.com/mcanouil/PRESENTATION/tree/master/julia_tp)
:::


## R Packages {data-icon=code}

```{r rpackages, results = "asis", eval = TRUE, echo = FALSE, message = FALSE, error = FALSE, warning = FALSE}
source("R/utils.R")
format_packages(package = source("R/packages.R")$value, author = "Mickaël Canouil", max = 57, output = "resume")
```


## Awards {data-icon=trophy}

### Funding Allocation SFD-Lilly

"Société Francophone du Diabète"  
(SFD)

Bordeaux, France

Mar. 2015

Project: *Detection of new genomic variants associated with fasting blood glucose and incidence of type 2 diabetes simultaneously*

::: aside
[sfdiabete.org](https://www.sfdiabete.org/medical/la-recherche/laureats-sfd/allocations-2015-les-laureats)  
[<i class="fa fa-github"></i> GitHub](https://github.com/mcanouil/PRESENTATION/tree/master/SFD2015)
:::


## Oral communications {data-icon=comment-dots}

### Jointly Modelling SNPs with Survival & Longitudinal Trait

[A Statistical Seminar Applied on Type 2 Diabetes](https://www.med.lu.se/kalendarium/190121_canouil/)  
(Host: Pr. Paul Franks)

Lund, Sweden

Jan. 2019

::: aside
[<i class="fa fa-github"></i> GitHub](https://github.com/mcanouil/PRESENTATION/tree/0b36aa0c679e07ffb448bb524ca06a20fd5779ae/joint_model)
:::

### Jointly Modelling SNPs with Survival & Longitudinal Trait

Thematic Day "Statistic & Genomic"  
of the "Réseau Interdisciplinaire  
autour de la Statistique"  
(RIS)

Paris, France

Oct. 2018

::: aside
[<i class="fa fa-github"></i> GitHub](https://github.com/mcanouil/PRESENTATION/tree/dd43462ae2e8a21243296c175ddccc2c1887b3ad/joint_model)
:::

### Longitudinal Genetic Modelling: Revisiting Associations of SNPs Associated with Blood Fasting Glucose in Normoglycemic Individuals

Statistical Methods for Post Genomic Data  
(SMPGD)

Lille, France

Feb. 2016

::: aside
[<i class="fa fa-github"></i> GitHub](https://github.com/mcanouil/PRESENTATION/tree/master/SMPGD2016)
:::


## Poster communications {data-icon=file}

### "Variants Génétiques Associés à la Trajectoire de la Glycémie à Jeun et à l’Incidence du Diabète de Type 2: Une Approche par Modèle Joint" (CA-075)

Annual Congress of  
"Société Francophone du Diabète"  
(SFD)

Lille, France

Mar. 2017

::: aside
[<i class="fa fa-github"></i> GitHub](https://github.com/mcanouil/PRESENTATION/tree/master/SFD2017)
:::

### Single Nucleotide Polymorphisms Associated With Fasting Blood Glucose Trajectory And Type 2 Diabetes Incidence: A Joint Modelling Approach

4th Symposium  
European Genomic Institute for Diabetes  
(EGID)

Lille, France

Nov. 2016

### Single Nucleotide Polymorphisms Associated With Fasting Blood Glucose Trajectory And Type 2 Diabetes Incidence: A Joint Modelling Approach

International Genetic Epidemiology Society  
(IGES)

Toronto, Canada

Oct. 2016

::: aside
[<i class="fa fa-github"></i> GitHub](https://github.com/mcanouil/PRESENTATION/tree/master/IGES2016)
:::

### Application of Joint Models in Genetic Association Studies

International Genetic Epidemiology Society  
(IGES)

Baltimore, United-States

Oct. 2015

::: aside
[<i class="fa fa-github"></i> GitHub](https://github.com/mcanouil/PRESENTATION/tree/master/IGES2015)
:::


<!-- ## Publications {data-icon=newspaper} -->

```{r, results = "asis", eval = TRUE, echo = FALSE, message = FALSE, error = FALSE, warning = FALSE}
source("R/utils.R")
articles <- read_bib("bib/myarticles.bib")
cat("\n")
cat("## Publications (", format(nrow(articles), big.mark = ","), "){data-icon=newspaper}\n\n", sep = "")
cat("\n")
format_bib(bib = articles, author = "Mickaël Canouil", max = 10, output = "resume")
```

