---
title: "Mickaël Canouil - Curriculum Vitæ"
author: Mickaël Canouil
output:
  pagedown::html_resume:
    css: ["css/resume.css", "css/curriculum_vitae.css"]
    mathjax: null
---

```{r, echo = FALSE, eval = FALSE, results = "hide"}
file.copy(system.file("resources", "css", "resume.css", package = "pagedown"), "css/resume.css")
```

```{r, results = "hide", eval = TRUE, echo = FALSE, message = FALSE, error = FALSE, warning = FALSE}
read_bib <- function(path) {
  big_file <- paste(readLines(path), collapse = "")
  big_file <- unlist(strsplit(x = big_file, split = "@", fixed = TRUE))
  big_file <- big_file[nchar(big_file)!=0]
  
  read_article <- function(.x) {
    clean_field <- function(pattern, x) {
      gsub(
        pattern = paste0("^", pattern, " = "), 
        replacement = "", 
        x = gsub(
          pattern = ",$", 
          replacement = "", 
          x = gsub(
            pattern = "[{}]", 
            replacement = "", 
            x = grep(paste0("^", pattern), x, value = TRUE)
         )
        )
      )
    }
    authors <- do.call("rbind", strsplit(unlist(strsplit(clean_field("author", .x), " and ")), ", "))
    authors <- gsub(" ", "&nbsp;", paste(authors[, 2], authors[, 1]))
    authors <- paste(paste(authors[-length(authors)], collapse = ", "), authors[length(authors)], sep = " and ")
    data.frame(
      title = clean_field("title", .x),
      month = gsub("May.", "May", paste0(Hmisc::capitalize(clean_field("month", .x)), ".")),
      year = clean_field("year", .x),
      doi = clean_field("doi", .x),
      authors = authors, 
      journal = clean_field("journal", .x),
      first = clean_field("first", .x),
      stringsAsFactors = FALSE
    )
  }
  
  all_bib <- lapply(strsplit(x = big_file, split = "\t"), read_article)
  all_bib <- do.call("rbind.data.frame", all_bib)
  all_bib[["month"]] <- factor(
    x = all_bib[["month"]],
    levels = gsub("May.", "May", paste0(c(
      "Jan", "Feb", "Mar", "Apr", "May", "Jun", 
      "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"
    ), "."))
  )
  all_bib[["doi"]] <- ifelse(
    test = grepl("^http", all_bib[["doi"]]), 
    yes = all_bib[["doi"]], 
    no = paste0("https://www.doi.org/", all_bib[["doi"]])
  )
  
  all_bib[order(all_bib[["year"]], all_bib[["month"]], decreasing = TRUE), ]
}

format_bib <- function(bib, author) {
  author <- gsub(" ", "&nbsp;", author)
  for (ipub in seq_len(nrow(bib))) {
    cat("\n")
    cat("###", bib[ipub, "title"], "\n\n")
    cat("\n")
    split_authors <- unlist(strsplit(strsplit(bib[["authors"]][ipub], ", ")[[1]], " and "))
    split_authors <- gsub(author, paste0("<u>", author, "</u>", if (bib[["first"]][ipub]) "<sup>†</sup>" else ""), split_authors)
    pos_author <- grep(author, split_authors)
    if (length(split_authors) > 10) {
      if (pos_author <= 10) {
        cat(
          paste(split_authors[1:10], collapse = ", "), 
          "*et&nbsp;al.*"
        )
      } else {
        cat(
          paste(c(split_authors[1:9], "*...*, "), collapse = ", "), 
          paste0(grep(pattern = author, x = split_authors, value = TRUE), "<sup>", pos_author, "</sup>"),
          "*et&nbsp;al.*"
        )
      }
    } else {
      cat(
        paste(split_authors[-length(split_authors)], collapse = ", "), 
        split_authors[length(split_authors)], 
        sep = " and "
      )
    }
    cat("\n")
    cat("\n")
    cat("N/A\n\n")
    cat("\n")
    cat(paste(bib[ipub, "month"], bib[ipub, "year"]), "\n\n")
    cat("\n")
    cat("::: aside\n")
    cat("*[", bib[ipub, "journal"],"](",  bib[ipub, "doi"], ")*\n", sep = "")
    if (bib[["first"]][ipub]) {
      cat("\n")
      cat("<sup>†</sup> As first or co-first author.\n")
      cat("\n")
    }
    cat(":::\n")
    cat("\n")
  }
}
```


# Aside

<!-- ![Mickaël Canouil](https://avatars3.githubusercontent.com/u/8896044){width=80%} -->
![Mickaël Canouil](pictures/id_picture.png){width=240px}


<!-- > The best thing about being a statistician is that you get to play in everyone’s backyard." -->
<!-- > -- John Tukey -->


## Contact Info {#contact}

- <i class="fa fa-user"></i> Head of the Biostatistic Team
- <i class="fa fa-university"></i> [EGID - UMR 8199](http://www.good.cnrs.fr/?lang=en)
- <i class="fa fa-map-marker"></i> Lille, France
- <i class="fa fa-envelope"></i> [mickael.canouil [at] cnrs.fr](mailto:mickael.canouil@cnrs.fr)
- <i class="fa fa-github"></i> [github.com/mcanouil](https://github.com/mcanouil)
- <i class="fa fa-phone"></i> +33 (0) 374 00 81 29


## Computer Skills {#skills}

- *Environment*: Docker, UNIX, Windows

- *Advanced*: R (Shiny, Rmarkdown, S4, etc.), HTML, css, LATEX, markdown

- *Intermediate*: bash, Julia, Perl, Python, SAS

- *Basic*: C/C++, SQL, NoSQL, Lua

## Disclaimer {#disclaimer}

<!-- Resume generated by Mickaël Canouil with the R package [**pagedown**](https://github.com/rstudio/pagedown). -->

Last updated on `r gsub("May.", "May", format(Sys.Date(), "%B. %Y"))`.



# Main

## Mickaël Canouil {#title}

### Curriculum Vitæ

I'm currently working for the *Institut Pasteur de Lille* 
at the *UMR 8199 - "Integrated Genomics and Metabolic Diseases Modelling"* 
as the *head of the biostatistic team*.

<!-- #### Main Activities -->

<!-- ##### Data Analysis -->

<!-- I've been analysing omics (SNP, CpG, expression, metabolites, etc.)  -->
<!-- data within numerous projects related to metabolic diseases,  -->
<!-- such as Type 2 Diabetes.  -->
<!-- These projects involve collaborations with national  -->
<!-- and international consortia like CKDgen, CHARGE, IMIDIA,  -->
<!-- DIRECT or more recently RHAPSODY. -->
<!-- My contribution covers Genome-Wide Association Studies (GWAS),  -->
<!-- differential methylation/expression analyses, metabolomics analyses,  -->
<!-- rare variants analyses (clustering approach), disease progression modelling,  -->
<!-- genetic epidemiology and meta-analyses. -->

<!-- ##### Team Management -->

<!-- I've been managing a team of three junior statisticians.  -->
<!-- My role is to provide guidance regarding choices of statistical methodologies  -->
<!-- for analysing the data and code optimisation for implementing these methodologies  -->
<!-- in large scale omics data. -->

<!-- ##### Research -->

<!-- My main research interests are mixed models, more recently extended to joint models. -->
<!-- Joint model, especially the joint likelihood approach implemented in the R package JM  -->
<!-- was studied in the context of Type 2 Diabetes incidence and fasting glucose progression  -->
<!-- (associated by diagnosis definition) using SNPs as biomarkers of interest. -->



## Education {data-icon=graduation-cap data-concise=true}

### Doctor of Philosophy (Ph.D.) in BioStatistics

University of Lille

Lille, France

Oct. 2014 - Sep. 2017

Thesis: *Development and Application of Statistical Methods for Multi-Omics Studies in Type 2 Diabetes: Beyond the Genome-Wide Association Studies Era*

### Master's Degree in Biostatistics, Bioinformatics and Genomics (*Specialised in Biostatistics*)

University Claude Bernard Lyon 1

Lyon, France

Sep. 2009 - Jul. 2011

Internships:

- *Mathematical Modelling of Nosocomial Rotavirus Infections in Pediatric Ward*
- *Development of a Non-Parametric (on Time) Clustering Algorithm for Longitudinal Data*

### Bachelor's Degree in Biology (*Specialised in Biostatistics*)

University Claude Bernard Lyon 1

Lyon, France

Sep. 2006 - Jul. 2009


## Professional & Research Experience {data-icon=laptop}

### Head of the Biostatistic Team

Laboratory of Integrated Genomics  
and Metabolic Diseases Modeling  
EGID - UMR 8199  
(European Genomics Institute for Diabetes)

Lille, France

Jan. 2016 - Present

Activities: *Genome-wide association studies, experimental design, -omics data analysis, methodological developments, team management*

### Biostatistician

Laboratory of Integrated Genomics  
and Metabolic Diseases Modeling  
EGID - UMR 8199  
(European Genomics Institute for Diabetes)

Lille, France

Sep. 2012 - Dec. 2015

Activities: *Genome-wide association studies, experimental design, -omics data analysis, methodological developments*

### Biostatistician

French Institute of Science and Technology for  
Transport, Development and Networks 
(IFSTTAR)  
UMRESTTE - UMR T9405

Bron, France

Nov. 2011 - Dec. 2011

Activities: *Analysis of a mobility and accident study in secondary-school pupils*

### Biostatistician (Internship)

Hospices Civiles de Lyon  
Biostatistics Unit  
UMR CNRS 5558

Lyon, France

Jan. 2011 - Jun. 2011

Activities: *Development of a Non-Parametric (on Time) Clustering Algorithm for Longitudinal Data*

### Biostatistician (Internship)

Biometry and Evolutionary Biology laboratory  
(LBBE)  
UMR CNRS 5558

Lyon, France

Mar. 2010 - Jun. 2010

Activities: *Mathematical Modelling of Nosocomial Rotavirus Infections in Pediatric Ward*


## Teaching Experience {data-icon=chalkboard-teacher}

### R and Databases

Two-days training (in French)  
"Unités Régionales de Formation  
à l'Information Scientifique et Technique"  
(URFIST)

Bordeaux, France

2018

::: aside
[<i class="fa fa-github"></i> GitHub](https://github.com/mcanouil/PRESENTATION/tree/master/rdatabase)
:::

### Julia for Intensive Scientific Computing

Half-day workshop (in French)  
"Journées nationales du  
DEVeloppement logiciel"  
(JDEV)

Bordeaux, France

2015

::: aside
[<i class="fa fa-github"></i> GitHub](https://github.com/mcanouil/PRESENTATION/tree/master/julia_tp)
:::


## R Packages {data-icon=code}

### CARoT: Centralised and Automated RepOrting Tools

Mickaël Canouil, Mathilde Boissel and Lijiao Ning 

N/A

Apr. 2019

[https://github.com/mcanouil/CARoT/](https://github.com/mcanouil/CARoT/)

::: aside
[![CRAN_Status_Badge](http://www.r-pkg.org/badges/version-ago/CARoT?color=brightgreen)](https://cran.r-project.org/package=CARoT)
:::

### NACHO: NanoString Quality Control Dashboard

Mickaël Canouil, Gerard Bouland and Roderick Slieker

N/A

Apr. 2019

[https://cran.r-project.org/package=NACHO](https://cran.r-project.org/package=NACHO)

::: aside
[![CRAN_Status_Badge](http://www.r-pkg.org/badges/version-ago/NACHO?color=brightgreen)](https://cran.r-project.org/package=NACHO)
:::

### clere: Simultaneous Variables Clustering and Regression

Loïc Yengo, Julien Jacques, Christophe Biernacki and Mickaël Canouil

N/A

Feb. 2014

[https://cran.r-project.org/package=clere](https://cran.r-project.org/package=clere)

::: aside
[![CRAN_Status_Badge](http://www.r-pkg.org/badges/version-ago/clere?color=brightgreen)](https://cran.r-project.org/package=clere)
:::

### snpEnrichment: SNPs Enrichment Analysis

Mickaël Canouil and Loïc Yengo

N/A

Apr. 2013

[https://cran.r-project.org/package=snpEnrichment](https://cran.r-project.org/package=snpEnrichment)

::: aside
[![CRAN_Status_Badge](http://www.r-pkg.org/badges/version-ago/snpEnrichment?color=brightgreen)](https://cran.r-project.org/package=snpEnrichment)
:::


## Awards {data-icon=trophy}

### Funding Allocation SFD-Lilly

"Société Francophone du Diabète  
(SFD)

Bordeaux, France

2015

Project: *Detection of new genomic variants associated with fasting blood glucose and incidence of type 2 diabetes simultaneously*

::: aside
[sfdiabete.org](https://www.sfdiabete.org/medical/la-recherche/laureats-sfd/allocations-2015-les-laureats)
:::


## Oral communications {data-icon=comment-dots}

### Jointly Modelling SNPs with Survival & Longitudinal Trait

Thematic Day "Statistic & Genomic"  
of the "Réseau Interdisciplinaire  
autour de la Statistique"  
(RIS)

Paris, France

2018

::: aside
[<i class="fa fa-github"></i> GitHub](https://github.com/mcanouil/PRESENTATION/tree/master/joint_model)
:::

### Longitudinal Genetic Modelling: Revisiting Associations of SNPs Associated with Blood Fasting Glucose in Normoglycemic Individuals

Statistical Methods for Post Genomic Data  
(SMPGD)

Lille, France

2016


## Poster communications {data-icon=file}

### "Variants Génétiques Associés à la Trajectoire de la Glycémie à Jeun et à l’Incidence du Diabète de Type 2: Une Approche par Modèle Joint" (CA-075)

Annual Congress of  
"Société Francophone du Diabète"  
(SFD)

Lille, France

2017

### Single Nucleotide Polymorphisms Associated With Fasting Blood Glucose Trajectory And Type 2 Diabetes Incidence: A Joint Modelling Approach

4th Symposium  
European Genomic Institute for Diabetes  
(EGID)

Lille, France

2016

### Single Nucleotide Polymorphisms Associated With Fasting Blood Glucose Trajectory And Type 2 Diabetes Incidence: A Joint Modelling Approach

International Genetic Epidemiology Society  
(IGES)

Toronto, Canada

2016

### Application of Joint Models in Genetic Association Studies

International Genetic Epidemiology Society  
(IGES)

Baltimore, United-States

2015


<!-- ## Publications {data-icon=newspaper} -->

```{r, results = "asis", eval = TRUE, echo = FALSE, message = FALSE, error = FALSE, warning = FALSE}
articles <- read_bib("./bib/myarticles.bib")
cat("\n")
cat("## Publications (", format(nrow(articles), big.mark = ","), "){data-icon=newspaper}\n\n", sep = "")
cat("\n")
format_bib(articles, "Mickaël Canouil")
```

