<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="generator" content="pandoc" />
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <meta name="author" content="Mickaël Canouil" />
    <meta name="date" content="2019-05-20" />
    <title>Mickaël Canouil</title>

        <link href="curriculum_vitae_files/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
    <link href="curriculum_vitae_files/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />
    <link href="curriculum_vitae_files/paged-0.2.6/css/resume.css" rel="stylesheet" />
    <script src="curriculum_vitae_files/paged-0.2.6/js/config.js"></script>
    <script src="curriculum_vitae_files/paged-0.2.6/js/paged.js"></script>
    <script src="curriculum_vitae_files/paged-0.2.6/js/hooks.js"></script>
    
    
  </head>
  <body>

<div id="aside" class="section level1">
<h1>Aside</h1>
<p><img src="https://avatars3.githubusercontent.com/u/8896044" alt="Mickaël Canouil" style="width:80.0%" /></p>
<!-- > The best thing about being a statistician is that you get to play in everyone’s backyard." -->
<!-- > -- John Tukey -->
<div id="contact" class="section level2">
<h2>Contact Info</h2>
<ul>
<li><i class="fa fa-user"></i> Head of the Biostatistic Team</li>
<li><i class="fa fa-university"></i> EGID - UMR 8199</li>
<li><i class="fa fa-map-marker"></i> Lille, France</li>
<li><i class="fa fa-envelope"></i> <a href="mailto:mickael.canouil@cnrs.fr" class="email">mickael.canouil@cnrs.fr</a></li>
<li><i class="fa fa-github"></i> <a href="https://github.com/mcanouil">github.com/mcanouil</a></li>
<li><i class="fa fa-phone"></i> +33 (0) 374 00 81 29</li>
<li>For more information, please contact me via email.</li>
</ul>
</div>
<div id="skills" class="section level2">
<h2>Computer Skills</h2>
<ul>
<li><p><em>Environment</em>: Docker, UNIX, Windows</p></li>
<li><p><em>Advanced</em>: R (Shiny, Rmarkdown, S4, etc.), HTML, css, LATEX, markdown</p></li>
<li><p><em>Intermediate</em>: bash, Julia, Perl, Python, SAS</p></li>
<li><p><em>Basic</em>: C/C++, SQL, NoSQL, Lua</p></li>
</ul>
</div>
<div id="disclaimer" class="section level2">
<h2>Disclaimer</h2>
<!-- Resume generated by Mickaël Canouil with the R package [**pagedown**](https://github.com/rstudio/pagedown). -->
<p>Last updated on May 2019.</p>
</div>
</div>
<div id="main" class="section level1">
<h1>Main</h1>
<div id="title" class="section level2">
<h2>Mickaël Canouil</h2>
<div id="curriculum-vit" class="section level3">
<h3>Curriculum Vitæ</h3>
<p>I’m currently working for the <em>Institut Pasteur de Lille</em>
at the <em>UMR 8199 - “Integrated Genomics and Metabolic Diseases Modelling”</em>
as the <em>head of the biostatistic team</em>.</p>
<!-- #### Main Activities -->
<!-- ##### Data Analysis -->
<!-- I've been analysing omics (SNP, CpG, expression, metabolites, etc.)  -->
<!-- data within numerous projects related to metabolic diseases,  -->
<!-- such as Type 2 Diabetes.  -->
<!-- These projects involve collaborations with national  -->
<!-- and international consortia like CKDgen, CHARGE, IMIDIA,  -->
<!-- DIRECT or more recently RHAPSODY. -->
<!-- My contribution covers Genome-Wide Association Studies (GWAS),  -->
<!-- differential methylation/expression analyses, metabolomics analyses,  -->
<!-- rare variants analyses (clustering approach), disease progression modelling,  -->
<!-- genetic epidemiology and meta-analyses. -->
<!-- ##### Team Management -->
<!-- I've been managing a team of three junior statisticians.  -->
<!-- My role is to provide guidance regarding choices of statistical methodologies  -->
<!-- for analysing the data and code optimisation for implementing these methodologies  -->
<!-- in large scale omics data. -->
<!-- ##### Research -->
<!-- My main research interests are mixed models, more recently extended to joint models. -->
<!-- Joint model, especially the joint likelihood approach implemented in the R package JM  -->
<!-- was studied in the context of Type 2 Diabetes incidence and fasting glucose progression  -->
<!-- (associated by diagnosis definition) using SNPs as biomarkers of interest. -->
</div>
</div>
<div id="education" class="section level2" data-icon="graduation-cap" data-concise="true">
<h2>Education</h2>
<div id="doctor-of-philosophy-ph.d.-in-biostatistics" class="section level3">
<h3>Doctor of Philosophy (Ph.D.) in BioStatistics</h3>
<p>University of Lille</p>
<p>Lille, France</p>
<p>Oct. 2014 - Sept. 2017</p>
<p>Thesis: Development and Application of Statistical Methods for Multi-Omics Studies in Type 2 Diabetes: Beyond the Genome-Wide Association Studies Era</p>
</div>
<div id="masters-degree-in-biostatistics-bioinformatics-and-genomics-specialised-in-biostatistics" class="section level3">
<h3>Master’s Degree in Biostatistics, Bioinformatics and Genomics (Specialised in Biostatistics)</h3>
<p>University Claude Bernard Lyon 1</p>
<p>Lyon, France</p>
<p>Sept. 2009 - Jul. 2011</p>
</div>
<div id="bachelors-degree-in-biology-specialised-in-biostatistics" class="section level3">
<h3>Bachelor’s Degree in Biology (Specialised in Biostatistics)</h3>
<p>University Claude Bernard Lyon 1</p>
<p>Lyon, France</p>
<p>Sept. 2006 - Jul. 2009</p>
</div>
</div>
<div id="professional-research-experience" class="section level2" data-icon="laptop">
<h2>Professional &amp; Research Experience</h2>
<div id="biostatistician-internship" class="section level3">
<h3>Biostatistician (Internship)</h3>
<p>Laboratory of Biometry and Evolutionary Biology<br />
(LBBE)<br />
UMR CNRS 5558</p>
<p>Lyon, France</p>
<p>Mar. 2010 - Jun. 2010</p>
<ul>
<li>Activities: Mathematical Modelling of Nosocomial Rotavirus Infections in Pediatric Ward</li>
</ul>
</div>
<div id="biostatistician-internship-1" class="section level3">
<h3>Biostatistician (Internship)</h3>
<p>Hospices Civiles de Lyon<br />
Biostatistics Unit<br />
UMR CNRS 5558</p>
<p>Lyon, France</p>
<p>Jan. 2011 - Jun. 2011</p>
<ul>
<li>Activities: Development of a Non-Parametric (on Time) Clustering Algorithm for Longitudinal Data</li>
</ul>
</div>
<div id="biostatistician" class="section level3">
<h3>Biostatistician</h3>
<p>The French Institute of Science<br />
and Technology for Transport,<br />
Development and Networks
(IFSTTAR)<br />
UMRESTTE - UMR T9405</p>
<p>Bron, France</p>
<p>Nov. 2011 - Dec. 2011</p>
<ul>
<li>Activities: Analysis of a mobility and accident study in secondary-school pupils</li>
</ul>
</div>
<div id="biostatistician-1" class="section level3">
<h3>Biostatistician</h3>
<p>Laboratory of Integrated Genomics<br />
and Metabolic Diseases Modeling<br />
EGID - UMR 8199<br />
(European Genomics Institute for Diabetes)</p>
<p>Lille, France</p>
<p>Sept. 2012 - Dec. 2015</p>
<ul>
<li>Activities: Genome-wide association studies, experimental design, -omics data analysis, methodological developments</li>
</ul>
</div>
<div id="head-of-the-biostatistic-team" class="section level3">
<h3>Head of the Biostatistic Team</h3>
<p>Laboratory of Integrated Genomics<br />
and Metabolic Diseases Modeling<br />
EGID - UMR 8199<br />
(European Genomics Institute for Diabetes)</p>
<p>Lille, France</p>
<p>Jan. 2016 - Present</p>
<ul>
<li>Activities: Genome-wide association studies, experimental design, -omics data analysis, methodological developments, team management</li>
</ul>
</div>
</div>
<div id="awards" class="section level2" data-icon="trophy">
<h2>Awards</h2>
<div id="funding-allocation-sfd-lilly" class="section level3">
<h3>Funding Allocation SFD-Lilly</h3>
<p>French speaking Diabetes Society (SFD)</p>
<p>Bordeaux, France</p>
<p>2015</p>
<ul>
<li>Detection of new genomic variants associated with fasting blood glucose and incidence of type 2 diabetes simultaneously</li>
</ul>
</div>
</div>
<div id="r-packages" class="section level2" data-icon="code">
<h2>R Packages</h2>
<div id="snpenrichment-snps-enrichment-analysis" class="section level3">
<h3>snpEnrichment: SNPs Enrichment Analysis</h3>
<p>Mickaël Canouil and Loïc Yengo</p>
<p>N/A</p>
<p>Apr. 2013</p>
<div class="aside">
<p><a href="https://cran.r-project.org/package=snpEnrichment">CRAN</a></p>
</div>
</div>
<div id="clere-simultaneous-variables-clustering-and-regression" class="section level3">
<h3>clere: Simultaneous Variables Clustering and Regression</h3>
<p>Loïc Yengo, Julien Jacques, Christophe Biernacki and Mickaël Canouil</p>
<p>N/A</p>
<p>Feb. 2014</p>
<div class="aside">
<p><a href="https://cran.r-project.org/package=clere">CRAN</a></p>
</div>
</div>
<div id="nacho-nacho-nanostring-quality-control-dashboard" class="section level3">
<h3>NACHO: NACHO: NanoString Quality Control Dashboard</h3>
<p>Mickaël Canouil, Gerard Bouland and Roderick Slieker</p>
<p>N/A</p>
<p>Apr. 2019</p>
<div class="aside">
<p><a href="https://cran.r-project.org/package=NACHO">CRAN</a></p>
</div>
</div>
<div id="carot-centralised-and-automated-reporting-tools" class="section level3">
<h3>CARoT: Centralised and Automated RepOrting Tools</h3>
<p>Mickaël Canouil, Mathilde Boissel and Lijiao Ning</p>
<p>N/A</p>
<p>Apr. 2019</p>
<div class="aside">
<p><a href="https://github.com/mcanouil/CARoT/">GitHub</a></p>
</div>
</div>
</div>
<div id="teaching-experience" class="section level2" data-icon="chalkboard-teacher">
<h2>Teaching Experience</h2>
<div id="julia-for-intensive-scientific-computing" class="section level3">
<h3>Julia for Intensive Scientific Computing</h3>
<p>Half-day workshop (in French)<br />
Journées nationales du DEVeloppement logiciel<br />
(JDEV)</p>
<p>Bordeaux, France</p>
<p>2015</p>
<div class="aside">
<p><a href="https://github.com/mcanouil/PRESENTATION/tree/master/julia_tp">GitHub</a></p>
</div>
</div>
<div id="r-and-databases" class="section level3">
<h3>R and Databases</h3>
<p>Two-days training (in French)<br />
URFIST<br />
University of Bordeaux</p>
<p>Bordeaux, France</p>
<p>2018</p>
<div class="aside">
<p><a href="https://github.com/mcanouil/PRESENTATION/tree/master/rdatabase">GitHub</a></p>
</div>
</div>
</div>
<div id="oral-communications" class="section level2" data-icon="comment-dots">
<h2>Oral communications</h2>
<div id="longitudinal-genetic-modelling-revisiting-associations-of-snps-associated-with-blood-fasting-glucose-in-normoglycemic-individuals" class="section level3">
<h3>Longitudinal Genetic Modelling: Revisiting Associations of SNPs Associated with Blood Fasting Glucose in Normoglycemic Individuals</h3>
<p>Statistical Methods for Post Genomic Data<br />
(SMPGD)</p>
<p>Lille, France</p>
<p>2016</p>
</div>
<div id="jointly-modelling-snps-with-survival-longitudinal-trait" class="section level3">
<h3>Jointly Modelling SNPs with Survival &amp; Longitudinal Trait</h3>
<p>Thematic Day “Statistic &amp; Genomic”<br />
of the<br />
Réseau Interdisciplinaire autour de la Statistique<br />
(RIS)</p>
<p>Paris, France</p>
<p>2018</p>
<div class="aside">
<p><a href="https://github.com/mcanouil/PRESENTATION/tree/master/joint_model">GitHub</a></p>
</div>
</div>
</div>
<div id="poster-communications" class="section level2" data-icon="file">
<h2>Poster communications</h2>
<div id="application-of-joint-models-in-genetic-association-studies" class="section level3">
<h3>Application of Joint Models in Genetic Association Studies</h3>
<p>International Genetic Epidemiology Society<br />
(IGES)</p>
<p>Baltimore, United-States</p>
<p>2015</p>
</div>
<div id="single-nucleotide-polymorphisms-associated-with-fasting-blood-glucose-trajectory-and-type-2-diabetes-incidence-a-joint-modelling-approach" class="section level3">
<h3>Single Nucleotide Polymorphisms Associated With Fasting Blood Glucose Trajectory And Type 2 Diabetes Incidence: A Joint Modelling Approach</h3>
<p>International Genetic Epidemiology Society<br />
(IGES)</p>
<p>Toronto, Canada</p>
<p>2016</p>
</div>
<div id="single-nucleotide-polymorphisms-associated-with-fasting-blood-glucose-trajectory-and-type-2-diabetes-incidence-a-joint-modelling-approach-1" class="section level3">
<h3>Single Nucleotide Polymorphisms Associated With Fasting Blood Glucose Trajectory And Type 2 Diabetes Incidence: A Joint Modelling Approach</h3>
<p>4th Symposium<br />
European Genomic Institute for Diabetes<br />
(EGID)</p>
<p>Lille, France</p>
<p>2016</p>
</div>
<div id="variants-genetiques-associes-a-la-trajectoire-de-la-glycemie-a-jeun-et-a-lincidence-du-diabete-de-type-2-une-approche-par-modele-joint-ca-075" class="section level3">
<h3>“Variants Génétiques Associés à la Trajectoire de la Glycémie à Jeun et à l’Incidence du Diabète de Type 2: Une Approche par Modèle Joint” (CA-075)</h3>
<p>Annual Congress<br />
“Société Francophone du Diabète”<br />
(SFD)</p>
<p>Lille, France</p>
<p>2017</p>
</div>
</div>
<div id="publications" class="section level2" data-icon="newspaper">
<h2>Publications</h2>
<div id="variable-clustering-in-high-dimensional-linear-regression-the-r-package-clere" class="section level3">
<h3>Variable Clustering in High-Dimensional Linear Regression: The R Package clere</h3>
<p>Loïc Yengo, Julien Jacques, Christophe Biernacki and <u>Mickaël Canouil</u></p>
<p>N/A</p>
<p>Apr. 2016</p>
<div class="aside">
<p><a href="https://journal.r-project.org/archive/2016/RJ-2016-006/">The R Journal</a></p>
</div>
</div>
<div id="hepatic-dpp4-dna-methylation-associates-with-fatty-liver" class="section level3">
<h3>Hepatic DPP4 DNA Methylation Associates With Fatty Liver</h3>
<p>Christian Baumeier, Sophie Saussenthaler, Anne Kammel, Markus Jähnert, Luisa Schlüter, Deike Hesse, <u>Mickaël Canouil</u>, Stephane Lobbens, Robert Caiazzo, Violeta Raverdy et al.</p>
<p>N/A</p>
<p>Jan. 2017</p>
<div class="aside">
<p><a href="https://www.doi.org/10.2337/db15-1716">Diabetes</a></p>
</div>
</div>
<div id="decreased-stard10-expression-is-associated-with-defective-insulin-secretion-in-humans-and-mice" class="section level3">
<h3>Decreased STARD10 Expression Is Associated with Defective Insulin Secretion in Humans and Mice</h3>
<p>Gaelle R. Carrat, Ming Hu, Marie-Sophie Nguyen-Tu, Pauline Chabosseau, Kyle J. Gaulton, Martijn van de Bunt, Afshan Siddiq, Mario Falchi, Matthias Thurner, <u>Mickaël Canouil</u> et al.</p>
<p>N/A</p>
<p>Feb. 2017</p>
<div class="aside">
<p><a href="https://www.doi.org/10.1016/j.ajhg.2017.01.011">American Journal of Human Genetics</a></p>
</div>
</div>
<div id="relationship-between-salivarypancreatic-amylase-and-body-mass-index-a-systems-biology-approach" class="section level3">
<h3>Relationship between salivary/pancreatic amylase and body mass index: a systems biology approach</h3>
<p>Amélie Bonnefond, Loïc Yengo, Aurélie Dechaume, <u>Mickaël Canouil</u>, Maxime Castelain, Estelle Roger, Frédéric Allegaert, Robert Caiazzo, Violeta Raverdy, Marie Pigeyre et al.</p>
<p>N/A</p>
<p>Feb. 2017</p>
<div class="aside">
<p><a href="https://www.doi.org/10.1186/s12916-017-0784-x">BMC medicine</a></p>
</div>
</div>
<div id="expression-and-functional-assessment-of-candidate-type-2-diabetes-susceptibility-genes-identify-four-new-genes-contributing-to-human-insulin-secretion" class="section level3">
<h3>Expression and functional assessment of candidate type 2 diabetes susceptibility genes identify four new genes contributing to human insulin secretion</h3>
<p>Fatou K. Ndiaye, Ana Ortalli, <u>Mickaël Canouil</u>, Marlène Huyvaert, Clara Salazar-Cardozo, Cécile Lecoeur, Marie Verbanck, Valérie Pawlowski, Raphaël Boutry, Emmanuelle Durand et al.</p>
<p>N/A</p>
<p>Jun. 2017</p>
<div class="aside">
<p><a href="https://www.doi.org/10.1016/j.molmet.2017.03.011">Molecular Metabolism</a></p>
</div>
</div>
<div id="low-dose-exposure-to-bisphenols-a-f-and-s-of-human-primary-adipocyte-impacts-coding-and-non-coding-rna-profiles" class="section level3">
<h3>Low-dose exposure to bisphenols A, F and S of human primary adipocyte impacts coding and non-coding RNA profiles</h3>
<p>Marie Verbanck, <u>Mickaël Canouil<sup>†<sup></u>, Audrey Leloire, Véronique Dhennin, Xavier Coumoul, Loïc Yengo, Philippe Froguel and Odile Poulain-Godefroy</p>
<p>N/A</p>
<p>Jun. 2017</p>
<div class="aside">
<p><a href="https://www.doi.org/10.1371/journal.pone.0179583">PloS One</a></p>
<p><sup>†</sup> As first or co-first author.</p>
</div>
</div>
<div id="a-large-scale-multi-ancestry-genome-wide-study-accounting-for-smoking-behavior-identifies-multiple-significant-loci-for-blood-pressure" class="section level3">
<h3>A Large-Scale Multi-ancestry Genome-wide Study Accounting for Smoking Behavior Identifies Multiple Significant Loci for Blood Pressure</h3>
<p>Yun J. Sung, Thomas W. Winkler, Lisa de Las Fuentes, Amy R. Bentley, Michael R. Brown, Aldi T. Kraja, Karen Schwander, Ioanna Ntalla, Xiuqing Guo , <em>…</em>, <u>Mickaël Canouil</u><sup>94</sup> <em>et al.</em></p>
<p>N/A</p>
<p>Mar. 2018</p>
<div class="aside">
<p><a href="https://www.doi.org/10.1016/j.ajhg.2018.01.015">American Journal of Human Genetics</a></p>
</div>
</div>
<div id="refining-the-accuracy-of-validated-target-identification-through-coding-variant-fine-mapping-in-type-2-diabetes" class="section level3">
<h3>Refining the accuracy of validated target identification through coding variant fine-mapping in type 2 diabetes</h3>
<p>Anubha Mahajan, Jennifer Wessel, Sara M. Willems, Wei Zhao, Neil R. Robertson, Audrey Y. Chu, Wei Gan, Hidetoshi Kitajima, Daniel Taliun , <em>…</em>, <u>Mickaël Canouil</u><sup>68</sup> <em>et al.</em></p>
<p>N/A</p>
<p>Apr. 2018</p>
<div class="aside">
<p><a href="https://www.doi.org/10.1038/s41588-018-0084-1">Nature Genetics</a></p>
</div>
</div>
<div id="increased-hepatic-pdgf-aa-signaling-mediates-liver-insulin-resistance-in-obesity-associated-type-2-diabetes" class="section level3">
<h3>Increased Hepatic PDGF-AA Signaling Mediates Liver Insulin Resistance in Obesity Associated Type 2 Diabetes</h3>
<p>Amar Abderrahmani, Loïc Yengo, Robert Caiazzo, <u>Mickaël Canouil<sup>†<sup></u>, Stéphane Cauchi, Violeta Raverdy, Valérie Plaisance, Valérie Pawlowski, Stéphane Lobbens, Julie Maillet et al.</p>
<p>N/A</p>
<p>May. 2018</p>
<div class="aside">
<p><a href="https://www.doi.org/10.2337/db17-1539">Diabetes</a></p>
<p><sup>†</sup> As first or co-first author.</p>
</div>
</div>
<div id="jointly-modelling-single-nucleotide-polymorphisms-with-longitudinal-and-time-to-event-trait-an-application-to-type-2-diabetes-and-fasting-plasma-glucose" class="section level3">
<h3>Jointly Modelling Single Nucleotide Polymorphisms With Longitudinal and Time-to-Event Trait: An Application to Type 2 Diabetes and Fasting Plasma Glucose</h3>
<p><u>Mickaël Canouil<sup>†<sup></u>, Beverley Balkau, Ronan Roussel, Philippe Froguel and Ghislain Rocheleau</p>
<p>N/A</p>
<p>Jun. 2018</p>
<div class="aside">
<p><a href="https://www.doi.org/10.3389/fgene.2018.00210">Frontiers in Genetics</a></p>
<p><sup>†</sup> As first or co-first author.</p>
</div>
</div>
<div id="novel-genetic-associations-for-blood-pressure-identified-via-gene-alcohol-interaction-in-up-to-570k-individuals-across-multiple-ancestries" class="section level3">
<h3>Novel genetic associations for blood pressure identified via gene-alcohol interaction in up to 570K individuals across multiple ancestries</h3>
<p>Mary F. Feitosa, Aldi T. Kraja, Daniel I. Chasman, Yun J. Sung, Thomas W. Winkler, Ioanna Ntalla, Xiuqing Guo, Nora Franceschini, Ching-Yu Cheng , <em>…</em>, <u>Mickaël Canouil</u><sup>87</sup> <em>et al.</em></p>
<p>N/A</p>
<p>Jun. 2018</p>
<div class="aside">
<p><a href="https://www.doi.org/10.1371/journal.pone.0198166">PLOS ONE</a></p>
</div>
</div>
<div id="type-2-diabetesassociated-variants-of-the-mt2-melatonin-receptor-affect-distinct-modes-of-signaling" class="section level3">
<h3>Type 2 diabetes–associated variants of the MT2 melatonin receptor affect distinct modes of signaling</h3>
<p>Angeliki Karamitri, Bianca Plouffe, Amélie Bonnefond, Min Chen, Jonathan Gallion, Jean-Luc Guillaume, Alan Hegron, Mathilde Boissel, <u>Mickaël Canouil</u>, Claudia Langenberg et al.</p>
<p>N/A</p>
<p>Aug. 2018</p>
<div class="aside">
<p><a href="https://www.doi.org/10.1126/scisignal.aan6622">Sci. Signal.</a></p>
</div>
</div>
<div id="fine-mapping-type-2-diabetes-loci-to-single-variant-resolution-using-high-density-imputation-and-islet-specific-epigenome-maps" class="section level3">
<h3>Fine-mapping type 2 diabetes loci to single-variant resolution using high-density imputation and islet-specific epigenome maps</h3>
<p>Anubha Mahajan, Daniel Taliun, Matthias Thurner, Neil R. Robertson, Jason M. Torres, N. William Rayner, Anthony J. Payne, Valgerdur Steinthorsdottir, Robert A. Scott , <em>…</em>, <u>Mickaël Canouil</u><sup>35</sup> <em>et al.</em></p>
<p>N/A</p>
<p>Oct. 2018</p>
<div class="aside">
<p><a href="https://www.doi.org/10.1038/s41588-018-0241-6">Nature Genetics</a></p>
</div>
</div>
<div id="multi-ancestry-study-of-blood-lipid-levels-identifies-four-loci-interacting-with-physical-activity" class="section level3">
<h3>Multi-ancestry study of blood lipid levels identifies four loci interacting with physical activity</h3>
<p>Lifelines Cohort Study Lifelines Cohort Study, Tuomas O. Kilpeläinen, Amy R. Bentley, Raymond Noordam, Yun Ju Sung, Karen Schwander, Thomas W. Winkler, Hermina Jakupović, Daniel I. Chasman , <em>…</em>, <u>Mickaël Canouil</u><sup>75</sup> <em>et al.</em></p>
<p>N/A</p>
<p>Jan. 2019</p>
<div class="aside">
<p><a href="https://www.doi.org/10.1038/s41467-018-08008-w">Nature Communications</a></p>
</div>
</div>
<div id="multi-ancestry-genome-wide-association-study-of-lipid-levels-incorporating-gene-alcohol-interactions" class="section level3">
<h3>Multi-Ancestry Genome-Wide Association Study of Lipid Levels Incorporating Gene-Alcohol Interactions</h3>
<p>Paul S. de Vries, Michael R. Brown, Amy R. Bentley, Yun J. Sung, Thomas W. Winkler, Ioanna Ntalla, Karen Schwander, Aldi T. Kraja, Xiuqing Guo , <em>…</em>, <u>Mickaël Canouil</u><sup>81</sup> <em>et al.</em></p>
<p>N/A</p>
<p>Jan. 2019</p>
<div class="aside">
<p><a href="https://www.doi.org/10.1093/aje/kwz005">American Journal of Epidemiology</a></p>
</div>
</div>
<div id="laser-capture-microdissection-of-human-pancreatic-islets-reveals-novel-eqtls-associated-with-type-2-diabetes" class="section level3">
<h3>Laser capture microdissection of human pancreatic islets reveals novel eQTLs associated with type 2 diabetes</h3>
<p>Amna Khamis, <u>Mickaël Canouil<sup>†<sup></u>, Afshan Siddiq, Hutokshi Crouch, Mario Falchi, Manon von Bulow, Florian Ehehalt, Lorella Marselli, Marius Distler, Daniela Richter et al.</p>
<p>N/A</p>
<p>Mar. 2019</p>
<div class="aside">
<p><a href="https://www.doi.org/10.1016/j.molmet.2019.03.004">Molecular Metabolism</a></p>
<p><sup>†</sup> As first or co-first author.</p>
</div>
</div>
<div id="multi-ancestry-genome-wide-genesmoking-interaction-study-of-387272-individuals-identifies-new-loci-associated-with-serum-lipids" class="section level3">
<h3>Multi-ancestry genome-wide gene–smoking interaction study of 387,272 individuals identifies new loci associated with serum lipids</h3>
<p>Amy R. Bentley, Yun J. Sung, Michael R. Brown, Thomas W. Winkler, Aldi T. Kraja, Ioanna Ntalla, Karen Schwander, Daniel I. Chasman, Elise Lim , <em>…</em>, <u>Mickaël Canouil</u><sup>97</sup> <em>et al.</em></p>
<p>N/A</p>
<p>Apr. 2019</p>
<div class="aside">
<p><a href="https://www.doi.org/10.1038/s41588-019-0378-y">Nature Genetics</a></p>
</div>
</div>
<div id="a-multi-ancestry-genome-wide-study-incorporating-gene-smoking-interactions-identifies-multiple-new-loci-for-pulse-pressure-and-mean-arterial-pressure" class="section level3">
<h3>A multi-ancestry genome-wide study incorporating gene-smoking interactions identifies multiple new loci for pulse pressure and mean arterial pressure</h3>
<p>Yun Ju Sung, Lisa de las Fuentes, Thomas W. Winkler, Daniel I. Chasman, Amy R. Bentley, Aldi T. Kraja, Ioanna Ntalla, Helen R. Warren, Xiuqing Guo , <em>…</em>, <u>Mickaël Canouil</u><sup>95</sup> <em>et al.</em></p>
<p>N/A</p>
<p>Apr. 2019</p>
<div class="aside">
<p><a href="https://www.doi.org/10.1093/hmg/ddz070">Human Molecular Genetics</a></p>
</div>
</div>
</div>
</div>

<script>
(function() {
  var i, s, j, el, els;
  var create_el = function(type, className, content, isHTML) {
    var el = document.createElement(type);
    if (className) el.className = className;
    if (content) {
      if (isHTML) {el.innerHTML = content;} else {el.innerText = content;}
    }
    return el;
  }

  // replace h2 title with h1
  var title = document.getElementById('title').querySelector('h2');
  title.parentNode.replaceChild(create_el('h1', false, title.innerHTML, true), title);

  // add the class .aside to #aside
  el = document.getElementById('aside');
  if (el) el.className += ' aside';

  // tweak class names of sections, and add icons
  els = document.getElementById('main').querySelectorAll('.level2');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (i === 0 && el.id === 'title') continue;
    el.className += ' main-block';
    if (el.dataset['concise']) el.className += ' concise';
    s = el.dataset['icon'] || 'bookmark';
    el.firstElementChild.insertBefore(create_el('i', 'fa fa-' + s), el.firstElementChild.firstChild);
  }

  // tweak class names of blocks in sections
  els = document.getElementById('main').querySelectorAll('.level3');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (el.parentNode.id === 'title') continue;
    el.className += ' blocks';
    el.innerHTML = '<div class="date"></div>' + '<div class="decorator"></div>' +
      '<div class="details"><header></header><div></div></div>' + el.innerHTML;
    var ps = el.querySelectorAll('p');
    // move the date paragraph to the date div
    if (ps.length >= 3) {
      s = ps[2].innerText;
      s = s === 'N/A' ? [] : s.split(' - ');
      el.removeChild(ps[2]);
      for (j = 0; j < s.length && j < 2; j++) {
        el.children[0].appendChild(create_el('span', false, s[j]))
      }
    }
    // move title, description, location to the details div
    (function(h) {
      h.appendChild(el.children[3]);
      var p = el.children[3];  // description
      if (p.innerText !== 'N/A') {
        h.appendChild(create_el('span', 'place', p.innerHTML, true));
      }
      el.removeChild(p);
      p = el.children[3];  // location
      if (p.innerText !== 'N/A') {
        s = create_el('span', 'location', p.innerHTML, true);
        s.innerHTML = '<i class="fa fa-map-marker-alt"></i> ' + s.innerHTML;
        h.appendChild(s);
      }
      el.removeChild(p);
    })(el.children[2].firstElementChild);
    // move the rest of content in a block into the last div of the details div
    s = el.children[2];
    while (j = s.nextSibling) {
      if (j.className === 'aside') { s = j; continue; }
      el.children[2].lastElementChild.append(j);
    }
  }
})();
</script>


  </body>
</html>
