% pdflatex CVMC_En.tex; bibtex CVMC_En; pdflatex CVMC_En.tex; pdflatex CVMC_En.tex; 

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[11pt,a4paper,sans]{moderncv}
\moderncvstyle{casual}
\moderncvcolor{grey} 
\usepackage[margin=2.5cm,bmargin=3cm]{geometry} % Reduce document margins
\usepackage[utf8]{inputenc}
\setlength{\footskip}{40pt}
%\setlength{\hintscolumnwidth}{3cm} % Uncomment to change the width of the dates column
%\setlength{\makecvtitlenamewidth}{10cm} % For the 'classic' style, uncomment to adjust the width of the space allocated to your name



%----------------------------------------------------------------------------------------
%	NAME AND CONTACT INFORMATION SECTION
%----------------------------------------------------------------------------------------

\firstname{Mickaël} % Your first name
\familyname{CANOUIL} % Your last name

% All information in this block is optional, comment out any lines you don't need
\title{Biostatistician, \textit{Ph.D.}}
\address{EGID - UMR 8199 - Pôle Recherche - 1er étage Aile Ouest}{1, Place de Verdun}{59045 LILLE CEDEX FRANCE}
\phone{+33(0)3 74 00 81 29}
\email{mickael.canouil@cnrs.fr}
\homepage{http://mickael.canouil.fr}
\extrainfo{https://github.com/mcanouil}
\quote{The best thing about being a statistician is that you get to play in everyone's backyard \\
-- John Tukey}
\photo[80pt][1pt]{../pictures/id_picture.png}

%----------------------------------------------------------------------------------------


\begin{document}


%----------------------------------------------------------------------------------------
%	COVER LETTER
%----------------------------------------------------------------------------------------

% To remove the cover letter, comment out this entire block

% \recipient{}{} % Letter recipient
% \date{October 22\textsuperscript{nd}, 2018} % Letter date
% \opening{Dear } % Opening greeting
% \closing{Sincerely yours,\vspace{-2.5em}} % Closing phrase
% \enclosure[Attached]{Curriculum Vit\ae{}} % List of enclosed documents
% 
% \makelettertitle % Print letter title
% 
% I'm currently working at the CNRS UMR8199 - ``Integrated Genomics and Metabolic Diseases Modelling'' as the head of the biostatistic team.\\
% I started working as a ``Study Engineer'' in September 2012.
% In October 2014, I was given the opportunity to start a Ph.D. thesis on joint models applied to the genetic field.\\
% This was an opportunity to get back to the study of longitudinal data in which I am interested in since my internship in 2011 (``Development of a non-parametric (on time) clustering algorithm for longitudinal data'').
% 
% During my six years at the CNRS UMR8199, I developed R packages \cite{yengo_variable_2016}, web applications using the R package \textsc{Shiny} \cite{ndiaye_expression_2017, verbanck_low-dose_2017} and wrote reproducible scripts from the data to the scientific article, using R and \textsc{Rmarkdown} \cite{canouil_jointly_2018}.\\
% Recently, I wrote a complete \textsc{Rmarkdown} analysis plan for the European project ``RHAPSODY WP3 - Pre-Diabetes progression''.
% The analysis plan combined with the standard format (CDISC) used for the phenotypes was included in a \textsc{Docker} image.
% This allows to run the whole analysis on any cohorts' data within the same environment (Operating System, R version and R packages version).
% 
% Over the years, I improved my computing skills in different programming languages particularly R, with for example \textsc{S4} object, \textsc{Rmarkdown} and \textsc{Shiny}.
% This also includes code optimisation, version control using GIT and more recently code portability and environment control using Docker.
% 
% Working at XXXXXXX could be a great opportunity for me to continue to work on joint models.
% In particular for the high expertise showed by XXXXXXX in longitudinal and survival data analysis, with for instance, XXXXXXX and the work of XXXXXXX\\
% With the knowledge acquired during my Ph.D. on joint models and my computing skills,
% I believe joining your team could benefit both sides. \\
% Thank you for considering my application.
% I would be happy to provide any additional information you might need.
% 
% \vspace{0.5em}\par{
% \textit{\textbf{References:}}
% \vspace{0.5em}
% \footnotesize
% \begin{description}
%   \setlength{\itemsep}{0.5em}
%   \item[Ghislain ROCHELEAU] Assistant Professor in Genetics and Genomic Sciences
%     \begin{description}
%       \item[Location] Mount Sinai, New-York, United States
%       \item[Mail] ghislain.rocheleau@mssm.edu
%     \end{description}
%   \item[Loïc YENGO-DEMBOU] Senior Research Officer
%     \begin{description}
%       \item[Location] Institute for Molecular Bioscience, Brisbane, Australia
%       \item[Mail] l.yengodimbou@imb.uq.edu.au
%       \item[Phone] +61 7 334 62095
%     \end{description}
% \end{description}
% }\vspace{0em}
% 
% \makeletterclosing % Print letter signature
% \clearpage



%----------------------------------------------------------------------------------------
%	CURRICULUM VIAE
%----------------------------------------------------------------------------------------

\makecvtitle % Print the CV title



%----------------------------------------------------------------------------------------
%	WORK EXPERIENCE SECTION
%----------------------------------------------------------------------------------------
\section{Experience}
%\subsection{Vocational}

\cventry{Jan. 2016 -- Present}{Head of the Biostatistic Team (Institut Pasteur de Lille)}{%
    CNRS UMR8199 - Integrated Genomics and Metabolic Diseases Modeling}{Lille}{%
    Activities: Genome-wide association studies, experimental design, -omics data analysis, methodological developments, consortium work package lead analyst, team management}{Headed by Pr. P. Froguel.}

\cventry{Sept. 2012 -- Dec. 2015}{Biostatistician (CNRS)}{%
    CNRS UMR8199 - Integrated Genomics and Metabolic Diseases Modeling}{Lille}{%
    Activities: Genome-wide association studies, experimental design, -omics data analysis, methodological developments}{Headed by Pr. P. Froguel.}

\cventry{Nov. 2011 -- Dec. 2011}{Biostatistician}{%
    The French Institute of Science and Technology for Transport, Development and Networks (IFSTTAR) - UMRESTTE - UMR T 9405}{Bron}{%
    Activities: Analysis of a mobility and accident study in secondary-school pupils}{Supervised by Dr. M. Haddak.}

\cventry{Jan. 2011 -- Jun. 2011}{Biostatistician (Internship)}{%
    Hospices Civiles de Lyon - Biostatistics Unit - CNRS UMR5558}{Lyon}{%
    Activities: Development of a non-parametric (on time) clustering algorithm for longitudinal data}{Supervised by Pr. R. Ecochard et Dr. C. Genolini.}

\cventry{Mar. 2010 -- Jun. 2010}{Biostatistician (Internship)}{%
    Laboratory of Biometry and Evolutionary Biology (LBBE) - CNRS UMR5558}{Lyon}{%
    Activities: Mathematical modelling of nosocomial rotavirus infections in pediatric ward}{Supervised by Dr. C. Kribs-Zaleta.}


\pagebreak
%----------------------------------------------------------------------------------------
%	MAIN ACTIVITIES SECTION
%----------------------------------------------------------------------------------------
\section{Main Activities In The Last Job Position}
\cvitem{Data Analysis}{%
I've been analysing omics (SNP, CpG, expression, metabolites, etc.) data within numerous projects related to metabolic diseases, such as Type 2 Diabetes. 
These projects involve collaborations with national and international consortia like CKDgen, CHARGE, IMIDIA, DIRECT or more recently RHAPSODY.
My contribution covers Genome-Wide Association Studies (GWAS), differential methylation/expression analyses, metabolomics analyses, rare variants analyses (clustering approach), disease progression modelling, genetic epidemiology and meta-analyses.}
\cvitem{Team Management}{%
I've been managing a team of three junior statisticians. 
My role is to provide guidance regarding choices of statistical methodologies for analysing the data and code optimisation for implementing these methodologies in large scale omics data.}
\cvitem{Research}{%
My main research interests are \textbf{mixed models}, more recently extended to joint models.
\textbf{Joint model}, especially the \textbf{joint likelihood approach} implemented in the R package \textsc{JM} was studied in the context of Type 2 Diabetes incidence and fasting glucose progression (associated by diagnosis definition) using SNPs as biomarkers of interest \cite{canouil_jointly_2018}.}



%----------------------------------------------------------------------------------------
%	EDUCATION SECTION
%----------------------------------------------------------------------------------------
\section{Education}
\cventry{Oct. 2014 -- Sept. 2017}{Doctor of Philosophy (Ph.D.) in BioStatistics}{University of Lille 2}{Lille}{%
    ``Development and Application of Statistical Methods for Multi-Omics Studies in Type 2 Diabetes: Beyond the Genome-Wide Association Studies Era''}{%
    Supervised by Pr. P. Froguel and Dr. G. Rocheleau}
\cventry{Sept. 2009 -- Jul. 2011}{Master's Degree in Biostatistics, Bioinformatics and Genomics}{University Claude Bernard Lyon 1}{Lyon}{%
    Specialised in Biostatistics}{}
\cventry{Sept. 2006 -- Jul. 2009}{Bachelor's Degree in Biology}{University Claude Bernard Lyon 1}{Lyon}{%
    Specialised in Mathematics and Informatics for Biology}{}



%----------------------------------------------------------------------------------------
%	COMPUTER SECTION
%----------------------------------------------------------------------------------------
\section{Computer Skills}
\cvitem{Basic}{\textsc{C/C++}, \textsc{SQL}, \textsc{NoSQL}}
\cvitem{Intermediate}{\textsc{Julia}, \textsc{Lua}, \textsc{Perl}, \textsc{Python}, \textsc{SAS}}
\cvitem{Advanced}{\textsc{R} (Shiny, Rmarkdown, S4, etc.), \textsc{HTML}, \textsc{css}, \textsc{LATEX}, \textsc{markdown}}
\vspace{1em}
\cvitem{Environment}{\textsc{Docker}, \textsc{UNIX}, \textsc{Windows}}



%----------------------------------------------------------------------------------------
%	LANGUAGES SECTION
%----------------------------------------------------------------------------------------
\section{Languages}
\cvitemwithcomment{French}{Native}{}
\cvitemwithcomment{English}{Fluent / Full Professional Proficiency}{}
\cvitemwithcomment{Spanish}{Elementary proficiency}{}



%----------------------------------------------------------------------------------------
%	Awards
%----------------------------------------------------------------------------------------
\section{Awards}
\cventry{2015}{Funding Allocation SFD-Lilly}{French speaking Diabetes Society (SFD)}{Bordeaux}{Detection of new genomic variants associated with fasting blood glucose and incidence of type 2 diabetes simultaneously}{}


\pagebreak
%----------------------------------------------------------------------------------------
%	Extensions R
%----------------------------------------------------------------------------------------
\section{R Packages}
\cvitem{clere}{\textbf{R package implementing the CLERE methodology.}
    \newline Loïc Yengo, Julien Jacques, Christophe Biernacki and \underline{Mickaël Canouil} (2014)
    \newline https://cran.r-project.org/package=clere}
\cvitem{snpEnrichment}{\textbf{R package implementing a method for calculating an enrichment statistic of a set of SNP within a GWA signal using minor allele frequency and linkage disequilibrium.}
    \newline \underline{Mickaël Canouil} and Loïc Yengo (2013)
    \newline https://cran.r-project.org/package=snpEnrichment}
\cvitem{NACHO}{\textbf{NACHO: NanoString Quality Control Dashboard.}
    \newline \underline{Mickaël Canouil}, Gerard Bouland and Roderick Slieker (2019)
    \newline https://cran.r-project.org/package=NACHO}
\cvitem{CARoT}{\textbf{Centralised and Automated RepOrting Tools.}
    \newline \underline{Mickaël Canouil}, Mathilde Boissel and Lijiao Ning (2019)
    \newline https://github.com/mcanouil/CARoT}



%----------------------------------------------------------------------------------------
%	Communications
%----------------------------------------------------------------------------------------
\section{Communications}
\subsection{Oral Presentations}
\begin{itemize}
    \setlength{\itemsep}{0.5em}

    \item \textbf{Julia for Intensive Scientific Computing} (\textit{half-day workshop available on GitHub, in French})
        \newline \underline{Mickaël Canouil}
        \newline \textit{``Journées nationales du DEVeloppement logiciel'' - JDEV, Bordeaux, France (2015)}

    \item \textbf{Longitudinal Genetic Modelling: Revisiting Associations of SNPs Associated with Blood Fasting Glucose in Normoglycemic Individuals}
        \newline \underline{Mickaël Canouil}, Ghislain Rocheleau, Loïc Yengo and Philippe Froguel
        \newline \textit{Statistical Methods for Post Genomic Data - SMPGD, Lille, France (2016)}
    
    \item \textbf{R and Databases} (\textit{two-days training available on GitHub, in French})
        \newline \underline{Mickaël Canouil}
        \newline \textit{URFIST - University of Bordeaux, Bordeaux, France (2018)}
        
    \item \textbf{Jointly Modelling SNPs with Survival \& Longitudinal Trait?} (\textit{available on GitHub, in French})
        \newline \underline{Mickaël Canouil}
        \newline \textit{Thematic Day ``Statistic \& Genomic'' of the ``Réseau Interdisciplinaire autour de la Statistique'' - RIS, Paris, France (2018)}

\end{itemize}

\subsection{Poster Presentations}
\begin{itemize}
    \setlength{\itemsep}{0.5em}

    \item \textbf{Application of Joint Models in Genetic Association Studies}
        \newline Ghislain Rocheleau, \underline{Mickaël Canouil}, Loïc Yengo and Philippe Froguel
        \newline \textit{International Genetic Epidemiology Society - IGES, Baltimore, United-States (2015)}

    \item \textbf{Single Nucleotide Polymorphisms Associated With Fasting Blood Glucose Trajectory And Type 2 Diabetes Incidence: A Joint Modelling Approach}
        \newline \underline{Mickaël Canouil}, Philippe Froguel and Ghislain Rocheleau
        \newline \textit{International Genetic Epidemiology Society - IGES, Toronto, Canada (2016)}

    \item \textbf{Single Nucleotide Polymorphisms Associated With Fasting Blood Glucose Trajectory And Type 2 Diabetes Incidence: A Joint Modelling Approach}
        \newline \underline{Mickaël Canouil}, Philippe Froguel and Ghislain Rocheleau
        \newline \textit{4th Symposium European Genomic Institute for Diabetes (E.g.i.d), Lille, France (2016)}

    \item \textbf{``Variants Génétiques Associés à la Trajectoire de la Glycémie à Jeun et à l’Incidence du Diabète de Type 2: Une Approche par Modèle Joint''} (CA-075)
        \newline \underline{Mickaël Canouil}, Philippe Froguel and Ghislain Rocheleau
        \newline \textit{Annual Congress of ``Société Francophone du Diabète'' (SFD), Lille, France (2017)}

\end{itemize}


%----------------------------------------------------------------------------------------
%	Publications
%----------------------------------------------------------------------------------------
% \section{Publications}
\pagebreak
\bibliographystyle{apalike}
\bibliography{../bib/myarticles.bib}
\nocite{*}
% \nocite{%
% yengo_variable_2016,
% baumeier_hepatic_2017,
% bonnefond_relationship_2017,
% carrat_decreased_2017,
% ndiaye_expression_2017,
% verbanck_low-dose_2017,
% sung_large-scale_2018,
% mahajan_refining_2018,
% abderrahmani_increased_2018,
% canouil_jointly_2018,
% feitosa_novel_2018,
% karamitri_type_2018,
% mahajan_fine-mapping_2018%
% }

\par{\vspace{1em}\textit{%
As co-first author in the following articles:
\begin{itemize}
  \item \cite{ndiaye_expression_2017}
  \item \cite{verbanck_low-dose_2017}
  \item \cite{abderrahmani_increased_2018}
  \item \cite{khamis_laser_2019}
\end{itemize}
\vspace{1em}
Note: authors' lists have been shortened with ``et al.'' and thus, might not contain my lastname.
}}


%----------------------------------------------------------------------------------------
%	INTERESTS SECTION
%----------------------------------------------------------------------------------------
\section{Interests}
% \renewcommand{\listitemsymbol}{+~} % Changes the symbol used for lists
\cvlistitem{Movies / TV-Shows / Japanese Animation}
\cvlistitem{Data visualisation (``IMDbRating'' on GitHub)}
\cvlistitem{Board games}
\cvlistitem{Hiking}
\cvlistitem{Archery}
\cvlistitem{A little bit of reading (fanstastic novels, e.g., Robin Hobb)}
\cvlistitem{Escape games}
% \cvitem{}{}

\end{document}
